app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -11,6 +11,29 @@ def create
       render(json: education_benefits_claim)
     end
 
+    # TODO: This is hidden behind a flag in the routes, but it's duplicated here for some
+    # defense in depth. This functionality should not be included once EducationForm is
+    # released in production, but rather should be available in a dedicated admin interface
+    def show
+      (redirect_to root_path and return) unless (Rails.env.development? || (ENV['EDU_FORM_SHOW'] == 'true'))
+
+      form = EducationBenefitsClaim.find(params[:id])
+      txt = ::EducationForm::CreateDailySpoolFiles.new.format_application(form.open_struct_form)
+
+      if params[:format] == 'tiff'
+        text = Tempfile.new('edu_txt')
+        tiff = Tempfile.new('edu_tiff')
+        text.write(txt)
+        text.close
+        # we fully control these paths.
+        `convert -bordercolor red -border 3 -append -density 100 -font CourierNew text:#{text.path} #{tiff.path}.tiff`
```

Comment:
I left in this code that makes use of `convert` (provided by ImageMagick) even though it isn't available on the dev/staging/prod servers entirely because its handy for local development. Happy to remove it though!